src/posix.mak

```diff
@@ -280,6 +281,8 @@ TK_SRC = \
 	$(TK)/filespec.h $(TK)/mem.h $(TK)/list.h $(TK)/vec.h \
 	$(TK)/filespec.c $(TK)/mem.c $(TK)/vec.c $(TK)/list.c
 
+STRING_IMPORT_FILES = verstr.h SYSCONFDIR
```

Comment:
I agree with @yebblies 
makefiles are hard enough to understand without the dual use of the name.